frontend/src/js/app.js

```diff
@@ -13,5 +13,5 @@ angular
 		'toaster',
 		'angularTrix',
 		'angularMoment',
-		'ngclipboard'
+		'ngclipboard',
```

Comment:
Why there is a `,` with the last element in the array. I am sure this will cause error.